### PR TITLE
Update Fedora test setup instructions

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: moduleroot/CONTRIBUTING.md.erb [hunk 1]
```diff
@@ -135,9 +135,19 @@ Our puppet modules provide [`Gemfile`](./Gemfile)s which can tell a ruby
...
 or Gems, are required to build, develop, and test this software.
 
-Please make sure you have [bundler installed](https://bundler.io/#getting-started)
-on your system, then use it to install all dependencies needed for this project,
-by running
+**Prerequisites**
+1. Make sure you have [bundler installed](https://bundler.io/#getting-started)
+on your system. If you are using Fedora, you can get `bundler` using
+   ```shell
+     % sudo dnf install rubygem-bundler
```
Comment (ekohl) [inline moduleroot/CONTRIBUTING.md.erb]: I know `#` is root and `$` is a normal user but what's the `%` convention? I do see it's used throughout the file so I guess I never really read this.

Comment (evgeni) [reply moduleroot/CONTRIBUTING.md.erb]: My zsh also uses `%` for user ;)

File: moduleroot/CONTRIBUTING.md.erb [hunk 2]
```diff
@@ -135,9 +135,19 @@ Our puppet modules provide [`Gemfile`](./Gemfile)s which can tell a ruby
...
+   ```
+2. If you are using Fedora, you may need these additional packages
+    ```shell
+      % sudo dnf install -y ruby-devel redhat-rpm-config rubygem-bundler
```
Comment (evgeni) [inline moduleroot/CONTRIBUTING.md.erb]: Bundler is already installed in step 1?

Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: ack, removed it now

File: moduleroot/CONTRIBUTING.md.erb (hunk 1 above)
Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: yeah, I am new to `%` as well, but thank you @evgeni for the info.

@ekohl I think we mostly use `$` and `#` in all other documentation, I am happy to fix it in this doc in a separate issue.

Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: @ekohl Actually I saw forklift docs, we don't use either if not required. So I removed `%` from the places where it is not really required.  I replaced `%' with `$` in just one instance where it makes sense to show it.

File: moduleroot/CONTRIBUTING.md.erb [hunk 3]
```diff
@@ -135,12 +135,22 @@ Our puppet modules provide [`Gemfile`](./Gemfile)s which can tell a ruby
...
 or Gems, are required to build, develop, and test this software.
 
-Please make sure you have [bundler installed](https://bundler.io/#getting-started)
-on your system, then use it to install all dependencies needed for this project,
-by running
+**Prerequisites**
+1. Make sure you have [bundler installed](https://bundler.io/#getting-started)
+on your system. If you are using Fedora, you can get `bundler` using
+   ```shell
+     sudo dnf install rubygem-bundler
```
Comment (ekohl) [inline moduleroot/CONTRIBUTING.md.erb]: You can keep the indenting at the same level as the backticks. This is true for a few cases in this file, but to keep it brief I won't repeat the comment every time.

**Comment by sthirugn** [comment]
@ekohl thanks, updated.

File: moduleroot/CONTRIBUTING.md.erb [hunk 4]
```diff
@@ -135,12 +135,22 @@ Our puppet modules provide [`Gemfile`](./Gemfile)s which can tell a ruby
...
+   ```shell
+   sudo dnf install -y ruby-devel redhat-rpm-config
+   ```
+
+Now, go to the root directory of this project and use `bundler` to install all
+dependencies needed for this project by running
 
-```shell
-% bundle install
+```
```
Comment (ekohl) [inline moduleroot/CONTRIBUTING.md.erb]: I meant like this:

```suggestion
```console
```

Comment (sthirugn) [reply moduleroot/CONTRIBUTING.md.erb]: I was not sure if `console` will be supported in github or not. Looks like it is supported - https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5055

**Comment by sthirugn** [comment]
@ekohl updated now

**Pull request merged**